src/firewall/core/nftables.py

```diff
@@ -488,7 +487,6 @@ def build_default_tables(self):
         default_tables = []
         default_tables.append({"add": {"table": {"family": "inet",
                                                  "name": TABLE_NAME}}})
-        self.created_tables["inet"].append(TABLE_NAME)
```

Comment:
`self.policy_table_active = True` ?

Reply:
not sure about what I just said... here we only make up the plan to add the table (didn't execute it yet successfully). Optimally, we set the flag TRUE a bit later, when we are confident that the table exists.

Is this flag still useful?

Optimally, we just delete/add the table (which does nothing if the tables is already deleted/added). Like recent `nft destroy table` command.

I guess, the problem is that we append the "delete" command to a list of commands, and we cannot indicate that that command can fail. I thought the workaround for no "destroy" is "add+delete" together? (since "add" does not fail if the table already exists).

Reply:
we could add a "Nftables.has_capability()" method, so we can ask it whether "destroy" is supported. If yes, use it. Otherwise, do "add+delete".

Reply:
`policy_table_active` is used for the _policy_ table, i.e. `firewalld_policy_drop`. This code is for the main `firewalld` table.